content/en/docs/concepts/workloads/controllers/replicationcontroller.md

```diff
@@ -115,8 +115,14 @@ specifies an expression with the name from each pod in the returned list.
 ## Writing a ReplicationController Spec
```

Comment:
(optional)
This heading is increasingly misnamed. I'd be happy to also:
```suggestion
## Writing a ReplicationController manifest
```

Reply:
I personally sort of loathe the word "manifest" for this, but it seems to be what we use, so....ACK

Reply:
I like that we have some name for these things that's not “a YAML”. And, in an ecosystem with an arguably unhealthy number of nautical puns, it does kind of fit.